plugins/node/opentelemetry-instrumentation-mysql2/src/utils.ts

```diff
@@ -103,10 +103,9 @@ export function getDbStatement(
  * @returns SQL statement without variable arguments or SQL verb
  */
 export function getSpanName(query: string | Query | QueryOptions): string {
-  if (typeof query === 'object') {
-    return query.sql;
-  }
-  return query.split(' ')[0];
+  const rawQuery = typeof query === 'object' ? query.sql : query;
+  // Extract the SQL verb
+  return rawQuery.split(' ')[0];
```

Comment:
optional nit: since the `query` object is received from user and `query.sql` can potentially be anything, a bit more robust writing will be to use optional chaining to make sure we don't throw exception here: `rawQuery?.split?.(' ')[0];`

Reply:
👍